src/components/Search.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "../styles/search.css";
+import getImages from "../requests/getImages";
+
+const Search = ({ setSearchResults }) => {
+  const [value, setValue] = useState();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setSearchResults(await getImages(value));
+  };
+  return (
+    <div className="search">
+      <form className="search-form" onSubmit={handleSubmit}>
+        <input
+          className="search-input"
+          type="text"
+          onChange={(e) => setValue(e.target.value)}
+        />
+        <button className="search-btn" type="submit">
+          Go!
+        </button>
+      </form>
+    </div>
+  );
+};
+export default Search;
+
+Search.propTypes = {
+  setSearchResults: PropTypes.func,
+};
```

Comment:
```
};

Search.propTypes = {
  setSearchResults: PropTypes.func,
};

export default Search;
```

you should add a blank line after your component, and usually the export will be at the bottom of the file